### PR TITLE
chore(holesky): pin lighthouse image, change docs

### PR DESCRIPTION
**Pull request by thedevbirb**
Minor changes to reflect the new Lighthouse release that removes the `always-prefer-builder-payload` flag. The image is now pinned, and docs have been changed a bit.

The section on what flags to provide is a bit generic but it's okay for now. Ideally we should add a comprehensive guide on which flags to enable on all beacon clients/validators clients.

### CODE REVIEW

File: testnets/holesky/README.md
Comment (mempirate) [inline testnets/holesky/README.md:80]: Where is this flag defined?

**Review by mempirate — APPROVED** [review]

File: testnets/holesky/README.md
Comment (mempirate) [inline testnets/holesky/README.md]: Typo: should be `builder-fallback-disable-checks`

**Review by mempirate — CHANGES_REQUESTED** [review]

**Comment by thedevbirb** [comment]
@mempirate I have to re-request a review otherwise I cannot merge it -- can we loosen up the requirements for merging for us core devs?

**Review by mempirate — APPROVED** [review]

**Pull request merged**
